src/modules/task/internal/task-reader.ts

```diff
@@ -0,0 +1,40 @@
+import {
+  GetAllTaskParams,
+  GetTaskParams,
+  Task,
+  TaskNotFoundError,
+} from '../types';
+import TaskRepository from './store/task-repository';
+import TaskUtil from './task-util';
+
+export default class TaskReader {
+  public static async getTask(params: GetTaskParams): Promise<Task> {
+    const task = await TaskRepository.task.findOne({
+      _id: params.taskId,
+      account: params.accountId,
+      active: true,
+    });
+    if (!task) {
+      throw new TaskNotFoundError(params.taskId);
+    }
+    return TaskUtil.convertTaskDBToTask(task);
+  }
+
+  public static async getTasks(params: GetAllTaskParams): Promise<Task []> {
+    let page = 1;
```

Comment:
Do not hard code.

Reply:
@jjalan  sir, this is not hardcoded value, these are the default values, for example client may not send the page and size in request params, in that case these default values will work.